src/ResponseBodyHandler.cpp

```diff
@@ -195,6 +214,8 @@ std::string getDefaultErrorPage(statusCode statusCode)
 	switch (statusCode) {
 	case StatusOK:
 		return ("");
+	case StatusCreated:
+		return ("");
```

Comment:
clang-tidy(bugprone-branch-clone): Switch has 2 consecutive identical branches

--> also compiles if I remove the first return ("");

Reply:
Ah yes, a case fallthrough. Removed the first return

Reply:
Clarification: Fallthrough would be if there would be an additional action, e.g.
```
Case1: 
  std::cout << "Msg1"; 
  //no break, this falls through
Case2:
  std::cout << "Msg2";
  break;
Case3:
  std::cout << "Msg3";
  break;
```
Case1 would print "Msg1Msg2"

Compiler should warn if it's an unannotated Fallthrough (at least with gcc)